fabfile.py

```diff
@@ -0,0 +1,149 @@
+from fabric.decorators import task
+from fabric.api import run
+from fabric.api import cd
+from fabric.contrib.files import exists
+from fabric.context_managers import settings
+from utils import arch, open_dir, clone
+from utils import get_list_of_failed_test_cases
+from utils import get_known_issues
+from utils import filter_known_issues
+
+import config as config
+
```

Comment:
It's good practice to add  [docstrings](https://www.python.org/dev/peps/pep-0257/) to each module an dmethod